libvast/src/detail/posix.cpp

```diff
@@ -60,6 +60,43 @@ int uds_accept(int socket) {
   return fd;
 }
 
+uds_datagram_sender::~uds_datagram_sender() {
+  ::unlink(src_path.c_str());
+}
+
+caf::expected<uds_datagram_sender>
+uds_datagram_sender::make(const std::string& path) {
+  auto result = uds_datagram_sender{};
+  result.src_fd = ::socket(AF_UNIX, SOCK_DGRAM, 0);
+  if (result.src_fd < 0)
+    return caf::make_error(ec::system_error,
+                           "failed to obtain an AF_UNIX DGRAM socket: {}",
+                           ::strerror(errno));
+  // TODO: Consider creating this inside the database directory instead.
```

Comment:
This is a must, actually. On hardened setups this will not be writable.

Reply:
I replaced this with a temporary location that should even be available with a very hardened systemd unit.